package.json

```diff
@@ -271,7 +271,7 @@
     "@metamask/address-book-controller": "^3.1.7",
     "@metamask/announcement-controller": "^6.1.0",
     "@metamask/approval-controller": "^6.0.0",
-    "@metamask/assets-controllers": "patch:@metamask/assets-controllers@patch%3A@metamask/assets-controllers@npm%253A26.0.0%23~/.yarn/patches/@metamask-assets-controllers-npm-26.0.0-17c0e9432c.patch%3A%3Aversion=26.0.0&hash=cf1d54#~/.yarn/patches/@metamask-assets-controllers-patch-0f46262fea.patch",
+    "@metamask/assets-controllers": "patch:@metamask/assets-controllers@npm%3A26.0.0#~/.yarn/patches/@metamask-assets-controllers-npm-26.0.0-17c0e9432c.patch",
```

Comment:
I don't know if this is right.  I did `yarn patch @metamask/assets-controllers` like usual.  But idk if you need to do something different to patch on top of a patch.

Reply:
That was automatically generated with `yarn patch @metamask/assets-controllers --update`
Can hop on a call to check this; i have another ongoing PR that patches same controller

Reply:
resolved offline